osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -210,29 +212,29 @@ protected sealed override void ApplyResultInternal(JudgementResult result)
 
             scoreResultCounts[result.Type] = scoreResultCounts.GetValueOrDefault(result.Type) + 1;
 
-            if (!result.Type.IsScorable())
-                return;
-
-            if (result.Type.IncreasesCombo())
-                Combo.Value++;
-            else if (result.Type.BreaksCombo())
-                Combo.Value = 0;
-
-            result.ComboAfterJudgement = Combo.Value;
-
-            if (result.Type.AffectsAccuracy())
+            if (maxResult.AffectsAccuracy())
             {
-                currentMaximumBaseScore += Judgement.ToNumericResult(result.Judgement.MaxResult);
+                currentMaximumBaseScore += Judgement.ToNumericResult(maxResult);
                 currentBaseScore += Judgement.ToNumericResult(result.Type);
                 currentAccuracyJudgementCount++;
             }
 
-            if (result.Type.IsBonus())
-                currentBonusPortion += GetBonusScoreChange(result);
-            else
-                currentComboPortion += GetComboScoreChange(result);
+            if (!result.Type.IsScorable())
```

Comment:
Is this condition reversed, or am I misunderstanding this?

Reply:
Yeah I'm a bit confused how this happened. It should be inverse (I likely broke this in refactoring after actually testing this PR).